src/rust/engine/src/nodes.rs

```diff
@@ -57,11 +57,17 @@ impl Context {
    */
   fn get<N: Node>(&self, node: N) -> NodeFuture<N::Output> {
     // TODO: Odd place for this... could do it periodically in the background?
-    maybe_drain_handles().map(|handles| {
-      externs::drop_handles(handles);
-    });
+    if N::is_inline() {
```

Comment:
Hm... this isn't returning anything... I don't think this will compile?

Also, the indentation in this method is strange.